src/sas/qtgui/Utilities/GuiUtils.py

```diff
@@ -411,7 +411,7 @@ def createModelItemWithPlot(update_data, name=""):
 
     checkbox_item = HashableStandardItem()
     checkbox_item.setCheckable(True)
-    checkbox_item.setCheckState(QtCore.Qt.Checked)
+    checkbox_item.setCheckState(QtCore.Qt.Unchecked)
```

Comment:
This solves the immediate issue.
However, maybe we can extend this method to allow for creation of checked items?
If we add an optional argument to the arg list and make the default `Unchecked`, then we can later call this method with this argument set if we ever need a checked item.
`def createModelItemWithPlot(update_data, name="", is_checked=False)`
...
`checkbox_item.setCheckState(QtCore.Qt.Checked if is_checked else QtCore.Qt.Unchecked)`

Reply:
True .. but can we think of a use case where `createModelItemWithPlot` should ever be used with `checked=True`?  I have thought about it but have yet to think of a case where that would be the right answer.  I figured if there is some very rare case that we cannot think of now, you can still do the same by adding the line
``` python
     new_item.setCheckState(QtCore.Qt.Checked)
```

My naive thinking is that it would be best not to add complexity to a function that will likely never be used? but am happy to have programming experts tell me otherwise.  I am a complete white belt in code design :-)

Reply:
> but have yet to think of a case where that would be the right answer

Then it's moot. The only reason for my comment was if there would be a case for it. I agree that making the function more complex is not sensible, especially if making it work with a very small additional effort is possible.